lib/matplotlib/ticker.py

```diff
@@ -2154,6 +2156,15 @@ def is_decade(x, base=10, *, rtol=1e-10):
     return is_close_to_int(lx, atol=rtol)
 
 
+def _is_decade(x, *, base=10, rtol=1e-5):
```

Comment:
1e-5 is numpy's default tolerance: https://numpy.org/doc/stable/reference/generated/numpy.isclose.html

Reply:
Also, it would be nice to have a more general word for "decade".

Reply:
Can we set it to `None` and let numpy fallback to its defalut?

Reply:
I'm not too worried about "decade", I do not know a more general "power-of-the-base" word in English and feel I have heard "a decade base 8" used.   Given the way that `base` threads through the log code, I think decade makes the code more readable in the common case with only minor confusion with a non-standard base.

Reply:
Yes, makes sense to use the default fall back.

OK! I added a doc-string if nothing else. (And now realize that `_is_integer_power_of_base` is a possible name, but I agree that `_is_decade` is probably more clear, especially since the base if rarely given.)